roles/openshift_storage_nfs/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Install nfs-utils
+  yum:
+    pkg: nfs-utils
+    state: present
+
+- name: Ensure exports directory exists
+  file:
+    path: "{{ exports_dir }}"
+    state: directory
+
+- name: Ensure export directories exist
+  file:
+    path: "{{ exports_dir }}/{{ item }}"
+    state: directory
+    mode: 0777
+    owner: nfsnobody
+    group: nfsnobody
+  with_items:
+  - "{{ registry_volume }}"
+
+- name: Configure exports
+  template:
+    dest: /etc/exports
+    src: exports.j2
+  notify:
+  - restart nfs-server
+
+- name: Enable and start services
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+  register: start_result
+  with_items:
+  - nfs-server
+
+- name: Pause 30 seconds if services were just restarted
+  pause: seconds=30
+  when: "{{ True in (start_result.results
+                      | map(attribute='changed')
+                      | list) }}"
```

Comment:
We've started replacing these pauses with adding a conditional on the handlers...  
see: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/tasks/main.yml#L40-L42
and: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/handlers/main.yml#L2-L4